api/src/main/java/com/okta/idx/sdk/api/client/IDXAuthenticationWrapper.java

```diff
@@ -369,11 +369,10 @@ public AuthenticationResponse recoverPassword(String username) {
      * @return the list of AuthenticatorUIOptions
      */
     public List<AuthenticatorUIOption> populateForgotPasswordAuthenticatorUIOptions(
-            IDXClientContext idxClientContext) {
+            IDXClientContext idxClientContext) throws ProcessingException {
```

Comment:
We haven't been exposing this in other methods. I wonder if it's best to return a different object here. One that contains a list of errors, and the list of UI options.

Or maybe attach some extra data to our `AuthenticationResponse` object.

Reply:
addressed this.